File: app/src/main/java/com/example/vpnapp/MainActivity.java
package com.example.vpnapp;

import android.os.Bundle;
import android.widget.Button;
import androidx.appcompat.app.AppCompatActivity;

public class MainActivity extends AppCompatActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btn = findViewById(R.id.changeIpButton);
        btn.setOnClickListener(v -> {
            // Здесь должна быть логика смены IP через VPN API
            btn.setText("IP обновлён!");
        });
    }
}

File: app/src/main/java/com/example/vpngateswitcher/MainActivity.kt
package com.example.vpngateswitcher

import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnSwitch = findViewById<Button>(R.id.btnSwitch)
        btnSwitch.setOnClickListener {
            // Здесь будет логика смены VPN-сервера через VPNGate
            // Пока просто тестовый вывод
            btnSwitch.text = "Новый IP активирован"
        }
    }
}
